scripts/imgtool/main.py

```diff
@@ -416,7 +416,7 @@ def sign(key, public_key_format, align, version, pad_sig, header_size,
         pad = True
     img = image.Image(version=decode_version(version), header_size=header_size,
                       pad_header=pad_header, pad=pad, confirm=confirm,
-                      align=int(align), slot_size=slot_size,
+                      align = int(align) if align else 1, slot_size=slot_size,
```

Comment:
Could we revert this back to `align=int(align), slot_size=slot_size,`
and apply the change suggested on line 379?